packages/backend/src/config/discoveryConfig.ts

```diff
@@ -3,104 +3,155 @@ import { DiscoveryConfig } from '@l2beat/discovery'
+            groupBy: 'chainId',
+            onlyValue: true,
+          },
+          defaultAppConfig: {
+            type: 'stateFromEvent',
+            event: 'SetDefaultConfigForChainId',
+            returnParams: [
+              'chainId',
+              'inboundProofLib',
+              'inboundBlockConfirm',
+              'outboundProofType',
+              'outboundBlockConfirm',
+              'oracle',
+            ],
+            groupBy: 'chainId',
+            onlyValue: true,
+          },
+          defaultAdapterParams: {
+            type: 'stateFromEvent',
+            event: 'SetDefaultAdapterParamsForChainId',
+            returnParams: ['chainId', 'proofType', 'adapterParams'],
+            groupBy: 'chainId',
+            onlyValue: true,
+          },
+          inboundProofLibrary: {
+            type: 'stateFromEvent',
+            event: 'AddInboundProofLibraryForChain',
+            returnParams: ['chainId', 'lib'],
+            groupBy: 'chainId',
+            onlyValue: true,
+            multipleInGroup: true,
+            ignoreRelative: true,
+          },
+          supportedOutboundProof: {
+            type: 'stateFromEvent',
+            event: 'EnableSupportedOutboundProof',
+            returnParams: ['chainId', 'proofType'],
+            groupBy: 'chainId',
+            onlyValue: true,
+            multipleInGroup: true,
+          },
+          INBOUND_LIBRARIES: {
+            type: 'arrayFromOneEvent',
+            event:
+              'event AddInboundProofLibraryForChain(uint16 indexed chainId, address lib)',
+            valueKey: 'lib',
+            ignoreRelative: true,
+          },
         },
       },
     },
-  },
+  }
 }
 
-export const discoveryConfig = new DiscoveryConfig(rawConfig)
+const ethereumRawConfig = createConfigFromTemplate({
+  name: 'layerzero',
+  chain: ChainId.ETHEREUM,
+  initialAddresses: [
+    '0x4D73AdB72bC3DD368966edD0f0b2148401A178E2', // STG
+  ],
+  addresses: {
+    ultraLightNodeV2: '0x4D73AdB72bC3DD368966edD0f0b2148401A178E2',
+    endpoint: '0x66A71Dcef29A0fFBDBE3c6a460a3B5BC225Cd675',
+    stargateMultisig: '0x65bb797c2B9830d891D87288F029ed8dACc19705',
+    layerZeroRelayer: '0x902F09715B6303d4173037652FA7377e5b98089E',
+    layerZeroOracle: '0x5a54fe5234E811466D5366846283323c954310B2',
+    layerZeroMultisig: '0xCDa8e3ADD00c95E5035617F970096118Ca2F4C92',
+  },
+})
+
+const arbitrumRawConfig = createConfigFromTemplate({
+  name: 'layerzero',
+  chain: ChainId.ARBITRUM,
+  initialAddresses: [
+    '0x4D73AdB72bC3DD368966edD0f0b2148401A178E2', // STG
+  ],
+  addresses: {
+    ultraLightNodeV2: '0x4D73AdB72bC3DD368966edD0f0b2148401A178E2',
+    endpoint: '0x3c2269811836af69497E5F486A85D7316753cf62',
+    stargateMultisig: '0x65bb797c2B9830d891D87288F029ed8dACc19705', // FIXME: not sure
+    layerZeroRelayer: '0x177d36dbe2271a4ddb2ad8304d82628eb921d790', // FIXME: not sure
+    layerZeroOracle: '0x5a54fe5234E811466D5366846283323c954310B2', // FIXME: not sure
```

Comment:
We can get rid of those three (also on Ethereum)

Reply:
[chore: fix configuration & inteface naming](https://github.com/l2beat/lz-monitoring/pull/16/commits/87f49ca01c29da3a02163278563be390c7145f73)